compliance_checker/suite.py

```diff
@@ -35,8 +35,15 @@ def load_all_available_checkers(cls):
         for x in working_set.iter_entry_points('compliance_checker.suites'):
             try:
                 xl = x.load()
-
                 cls.checkers[xl.name] = xl
+                # set any specific version numbers or pass if we don't have
+                # the _supported_versions attribute
+                try:
+                    for ver in sorted(xl._supported_versions):
```

Comment:
Right now it's how I'm enumerating which versions we're supporting.  Since I broke this off from #240, it was the mechanism used.  Basically there are two major choices: create a new class for each new version of a particular standard, or pass along a version argument to the checker constructor.  I opted for the latter since most changes are fairly minor between versions currently.  Perhaps the former is what should have been done, or delegate the base constructor to the various versions.  I wouldn't be averse to a refactor at some point.